scss/_collapse.scss

```diff
@@ -24,22 +24,24 @@
       padding: .75rem 1.25rem;
       color: $dark-silver;
       text-align: left;
+      text-transform: none;
 
       &::after {
         position: absolute;
-        top: -4px;
-        right: 25px;
+        top: 2px;
+        right: 20px;
         display: inline-block;
+        // stylelint-disable font-family-no-missing-generic-family-keyword
+        font-family: "Material Icons Sharp";
+        // stylelint-enable font-family-no-missing-generic-family-keyword
         font-size: 2em;
-        content: "\221F";
-        transform: rotate(-45deg);
+        content: "\e5cf";
```

Comment:
We shouldn't do this

Reply:
We should add the actual svg here somehow.

Reply:
@trackleft Not sure if it helps, but we could use "expand_more" or whatever the icon name is from Google Material instead of the codepoint.

Reply:
I would also expect the Google conglomerate to have a pretty solid process to not mess up their content without ample warning.